.env.production.example

```
@@ -0,0 +1,32 @@
+NEXT_PUBLIC_SUPABASE_ANON_KEY=your-supabase-anon-key
+SUPABASE_SERVICE_ROLE_KEY=your-service-role-key
+
+# MFA Configuration
+MFA_ENABLED=true
+MFA_ISSUER="ScriptGenius"
+MFA_CODE_VALIDITY_SECONDS=30
+MFA_BACKUP_CODES_COUNT=10
+
+# Session Configuration
+SESSION_SECRET=your-secure-session-secret
+SESSION_MAX_AGE=86400 # 24 hours
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SESSION_MAX_AGE key should go before the SESSION_SECRET key

(UnorderedKey)